docs/docs/Getting-Started/getting-started-installation.md

```diff
@@ -1,97 +1,115 @@
 ---
-title: 📦 Installation
+title: Install Langflow
 sidebar_position: 1
 slug: /getting-started-installation
 ---
 
+You can deploy Langflow either locally or as a hosted service with [**Datastax Langflow**](#datastax-langflow).
 
+## Install Langflow Locally
 
-## **DataStax Langflow** {#a3b4bd8dec5a49ebbfca4828492133e9}
+Install Langflow locally with [pip](https://pypi.org/project/pip/) or [pipx](https://pipx.pypa.io/stable/installation/).
 
+### Prerequisites
 
----
-
-
-The easiest way to get started with Langflow is through the DataStax Cloud Service!
+* [Python 3.10 to 3.12](https://www.python.org/downloads/release/python-3100/) installed
+* [pip](https://pypi.org/project/pip/) or [pipx](https://pipx.pypa.io/stable/installation/) installed
+* Before installing Langflow, we recommend creating a virtual environment to isolate your Python dependencies with [venv](https://docs.python.org/3/library/venv.html) or [conda](https://anaconda.org/anaconda/conda)
 
+### Install Langflow
```

Comment:
Since the title of the overall page is "Install Langflow", you should use a different title here. Suggest using "Install Langflow with pip or pipx"